### ISSUE TITLE
gunzip -c chr*.fa.masked > hg19.fa

### ISSUE THREAD

**Issue by zhenzuo2** [slowkow/CENTIPEDE.tutorial#13]
Hello,

When I run 

```
gunzip -c chr*.fa.masked > hg19.fa
```

in [CENTIPEDE.tutorial](https://slowkow.github.io/CENTIPEDE.tutorial/) in Genomic sequence Section, I got the following error

```
gzip: chr10.fa.masked: not in gzip format
gzip: chr11.fa.masked: not in gzip format
gzip: chr11_gl000202_random.fa.masked: not in gzip format
...
```

Am I doing it wrong? Thanks.

**Comment by slowkow** [comment]
Yes, gzip will throw that error when you try to decompress (`gunzip`) a file that is not compressed.

You only need to decompress a file when it is compressed (it will have the `.gz` file extension).

**Comment by slowkow** [comment]
I think you caught a typo in my tutorial! Thanks for reporting it.

I haven't tested it, but I think the correct command is:

```
cat chr*.fa.masked > hg19.fa
```

Good luck! Let me know how it goes. Thanks again for reporting the problem.

**Comment by zhenzuo2** [comment]
Thank you so much for your prompt reply. Your tutorial is the best tutorial I can find online. It helps a lot for beginners like me. 

I also have a question about where to download ENCFF001UUQ.narrowPeak.gz and ENCFF001UUQ_gt8.narrowPeak.gz. You explained it in DNase-Seq data section but I am still not very sure where to find them. Thanks.

**Comment by slowkow** [comment]
It looks like the particular files I used in the tutorial have been archived.

You should find a different study that interests you:

https://www.encodeproject.org/search/?type=Experiment&assay_term_name=DNase-seq&replicates.library.biosample.donor.organism.scientific_name=Homo+sapiens

Here is [one possible example][1], showing the links to the BAM file and the narrowPeak bed file:

[1]: https://www.encodeproject.org/experiments/ENCSR449HOQ/

![2018-09-16_17-54-45](https://user-images.githubusercontent.com/209714/45601388-220bae00-b9da-11e8-952c-c0d600a07e3e.png)

**Comment by zhenzuo2** [comment]
Thank you a lot. It makes sense now. 

Best,

Zhen
